### ISSUE TITLE
configurable timezone

### ISSUE THREAD

**Issue by splazit** [SocialEngine/rancher-cron#19]
This is more of a feature request rather than issue.
Could you please consider expose a env variable or label so that the timezone can be passed in? 
Thanks,

**Comment by XshzX** [comment]
Is there already an easy solution or any other way to set a new timezone effectively ?
Thanks

**Comment by splazit** [comment]
Yeah, I had to extend this docker image to add the timezone. Please see this: https://hub.docker.com/r/splazit/rancher-cron/~/dockerfile/
